### PR TITLE
Central logger link

### PR DESCRIPTION
**Pull request by fjoubert**
@theunsa please merge

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -54,6 +54,7 @@
                 secondary: 'dark-secondary',
                 primaryButtons: 'dark-buttons'
             }])
+        .constant('SERVER_URL', window.location.host === 'localhost:8000' ? 'http://monctl.devf.camlab.kat.ac.za' : window.location.origin)
```
Comment (theunsa) [inline app/app.js]: Note: We need to think about a better way to handle development especially now that other team members are going to start doing GUI development.

File: app/app.js [hunk 2]
```diff
@@ -345,6 +346,11 @@
                 });
         };
 
+        vm.openCentralLogger = function () {
+            //TODO get from config and eventually redo central logger
+            KatGuiUtil.openRelativePath('', 9021);
```
Comment (theunsa) [inline app/app.js]: Should this port number not also be a constant?

File: app/app.js (hunk 1 above)
Comment (fjoubert) [inline app/app.js]: Yeah we really should. I am moving the constant back to where it was because it broke too many tests.

File: app/app.js (hunk 2 above)
Comment (fjoubert) [inline app/app.js]: Good point, going to make it a constant.

**Comment by fjoubert** [comment]
@theunsa please merge if you are satisfied, would like to get jenkins happy again for katgui :)

**Pull request merged**